deduplicate/load-and-attach-r-packages.R

```diff
@@ -1,36 +1,34 @@
-suppressPackageStartupMessages({
-  library(assertthat)
```

Comment:
I moved this to inside `use_r_packages()`

Reply:
👍